recipes/cdat_info/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "cdat_info" %}
+{% set version = "2.8" %}
+#{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    git_rev: v2.8
+    git_url: git://github.com/UV-CDAT/uvcdat.git
```

Comment:
So you know,  I tried docker behind our firewall to your script `run_docker_build.sh`

I had to add my local certificate on Ubuntu for docker to load the image using `https`
```bash
* Copy CA cert to /usr/local/share/ca-certificates.
* sudo update-ca-certificates
* sudo service docker restart
```

As well, I changed you script and add the following conda command to theCONDARC section:
```
...
always_yes: true   
show_channel_urls: true    
**ssl_verify:false**
CONDARC  
)  
```